marshmallow_jsonapi/fields.py

```diff
@@ -71,6 +71,8 @@ def __init__(
         self.id_field = id_field or self.id_field
         super(Relationship, self).__init__(**kwargs)
         self.dump_only = kwargs.pop('dump_only', True)
+        if many and not self.dump_only:
+            raise ValueError('Loading is not supported for many=True.')
```

Comment:
I'm not sure about this constraint. Aren't there use cases where you would want to deserialize the a collection of objects passed in as IDs by the client?